projects/vdk-plugins/vdk-jobs-troubleshooting/src/vdk/plugin/jobs_troubleshoot/api/troubleshoot_utility.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+from abc import abstractmethod
+
+
+class ITroubleshootUtility:
+    """
+    Allows for troubleshoot utilities to be implemented. It provides a unified
+    API to be used by the jobs troubleshooting plugin irrespective of the
+    utility's underlying logic.
```

Comment:
I don't understand the logic with this sentence. What are you trying to say, that the API/contract is agnostic to impl..? That is common sense

Reply:
Changed the doc to be easier to understand. Thank you for the review